Stratis.Bitcoin.Features.BlockStore/BlockStoreBehavior.cs

```diff
@@ -93,7 +93,9 @@ protected override void DetachCore()
             this.logger.LogTrace("(-)");
         }
 
+#pragma warning disable IDE1006 // Naming Styles
         private async void AttachedNode_MessageReceived(Node node, IncomingMessage message)
+#pragma warning restore IDE1006 // Naming Styles
```

Comment:
why can't we just append `Async` or `_Async` here instead of using pragmas?

Reply:
I just didn't think it through.